library/ssl_tls13_generic.c

```diff
@@ -256,13 +256,14 @@ static int ssl_tls13_parse_certificate_verify( mbedtls_ssl_context *ssl,
 #if defined(MBEDTLS_X509_RSASSA_PSS_SUPPORT)
     if( sig_alg == MBEDTLS_PK_RSASSA_PSS )
     {
-        const mbedtls_md_info_t* md_info;
         rsassa_pss_options.mgf1_hash_id = md_alg;
-        if( ( md_info = mbedtls_md_info_from_type( md_alg ) ) == NULL )
+        psa_algorithm_t psa_alg = mbedtls_psa_translate_md( md_alg );
```

Comment:
Minor: I think this is already available as `hash_alg` and by this point we know it's not 0.